src/main.rs

```diff
@@ -86,8 +85,9 @@ struct CommonArgs {
 
     /// Static set of IP:Port where Shredstream proxy forwards shreds to, comma separated.
     /// Eg. `127.0.0.1:8002,10.0.0.1:8002`.
-    #[arg(long, env, value_delimiter = ',')]
-    dest_ip_ports: Vec<SocketAddr>,
+    // Note: store the original string so we can do hostname resolution when refreshing destinations
```

Comment:
does this comment make sense here?

Reply:
this is not exposed to end users.

help output:
```shell
      --dest-ip-ports <DEST_IP_PORTS>
          Static set of IP:Port where Shredstream proxy forwards shreds to, comma separated. Eg. `127.0.0.1:8002,10.0.0.1:8002` [env: DEST_IP_PORTS=]
      --endpoint-discovery-url <ENDPOINT_DISCOVERY_URL>
          Http JSON endpoint to dynamically get IPs for Shredstream proxy to forward shreds. Endpoints are then set-union with `dest-ip-ports` [env: ENDPOINT_DISCOVERY_URL=]
```